bedrock/mozorg/views.py

```diff
@@ -277,8 +277,9 @@ def mieco_email_form(request):
 
     email_to = MIECO_EMAIL_TO[form.cleaned_data.pop("message_id") or "mieco"]
     email_msg = render_to_string("mozorg/emails/mieco-email.txt", {"data": form.cleaned_data}, request=request)
+    email_sub = MIECO_EMAIL_SUBJECT[form.cleaned_data.pop("message_id") or "mieco"]
```

Comment:
Oh yeah, and you can't pop the `message_id` a 2nd time. You'll have to grab that above and set it to a variable, then use it in these two places.

Reply:
That's what was getting me!